src/sugar3/graphics/toolcombobox.py

```diff
@@ -41,13 +41,20 @@ class ToolComboBox(Gtk.ToolItem):
             Returns a new tool combobox.
     Keyword Args:
         combo:
+            The combo attribute is like a Gtk.ComboBox.
+            There are two types: Either user passed or
+            automatic.
```

Comment:
That's confusing.

When used as a "Keyword Args", `combo` is an argument, not an attribute.  `combo` must be either None or a ComboBox.  If None a new ComboBox will be created.

When a new ToolComboBox is instantiated, `combo` is an attribute, which will be a ComboBox, always.

Also, you say immediately below "*List of items in ToolComboBox It can also be set to none.*" and it is indented as part of Keyword Args.  There's no such Keyword Arg.